template/lambda-sqs-worker-cdk/.buildkite/pipeline.yml

```diff
@@ -56,12 +56,13 @@ steps:
     label: 🧪 Test, Lint & Build
     artifact_paths: lib/**/*
     commands:
+      - yarn package
```

Comment:
doesn't feel right to have to run yarn package here when it's already in Dockerfile

Reply:
Ah this isn't working because we've forgotten to add the docker ecr cache plugins here. Can you try adding the following plugin section to the step?

```
  plugins:
        - *aws-sm
        - *private-npm
        - *docker-ecr-cache
        - docker-compose#v3.9.0:
            dependencies: false
            run: app
```

Reply:
Does your test step require you to run yarn build beforehand?

Reply:
Oh I see what might be happening here. Do we need to mount lib/node_modules as a volume too? @72636c

Reply:
Could you try something for me - could you add
`- /workdir/lib/node_modules` to the volumes section in your docker-compose file and undo all changes except the deploy step in your repo?

Reply:
💡 That sounds right - it looks like `lib/node_modules` from `yarn package` is never mounted.

TBH I'm questioning why `yarn package` is even in the Dockerfile, it sounds like it belongs on the deploy step. I believe that was the rationale behind it being `--offline`, ideally it would be runnable without requiring access to the npm read token.

Reply:
Ooo that might be due to yarn cache clean afaik we need the cache to run any with --offline

Reply:
Curious, is the command doing anything that `cp -R` doesn't?

Reply:
Not sure aha, I think it may be possibly installing production node_modules in lib whilst leaving the Dev deps in tact? That's my theory

Reply:
Ah of course, thanks

Reply:
this worked, thanks!